recipe/dynamic-lookup.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/PyQt5_gpl-5.9.2/configure.py b/PyQt5_gpl-5.9.2/configure.py
+index a263efb..115075f 100644
+--- a/PyQt5_gpl-5.9.2/configure.py
++++ b/PyQt5_gpl-5.9.2/configure.py
+@@ -943,6 +943,9 @@ class TargetConfiguration:
+         Python interpreter library.
+         """
+ 
++        if self.py_platform == 'darwin':
++            return '-undefined dynamic_lookup'
++
+         args = qmake_quote('-L' + self.py_pylib_dir)
+ 
```

Comment:
Can you add a `return args` here for `linux`? We shouldn't link to the python shared library as python is statically linked.

Reply:
Could you post a new patch? I've found that modifying patches by hand usually doesn't work.

Reply:
```diff
diff --git a/PyQt5_gpl-5.9.2/configure.py b/PyQt5_gpl-5.9.2/configure.py
index a263efb..d244533 100644
--- a/PyQt5_gpl-5.9.2/configure.py
+++ b/PyQt5_gpl-5.9.2/configure.py
@@ -943,8 +943,14 @@ class TargetConfiguration:
         Python interpreter library.
         """
 
+        if self.py_platform == 'darwin':
+            return '-undefined dynamic_lookup'
+
         args = qmake_quote('-L' + self.py_pylib_dir)
 
+        if self.py_platform.startswith('linux'):
+            return args
+
         if name:
             args += ' -l' + self.py_pylib_lib
 ```

Reply:
Done!